### PR TITLE
Admixer Bid Adapter add consent support

### PR DESCRIPTION
**Pull request by Galphimbl**
## Type of change

- [x] Feature 

## Description of change

add gdpr consent and usp consent to bid request params

- A link to a PR on the docs repo [#1950](https://github.com/prebid/prebid.github.io/pull/1950)

### CODE REVIEW

File: integrationExamples/gpt/gdpr_hello_world.html [hunk 1]
```diff
@@ -82,37 +82,37 @@
       <script>
         var PREBID_TIMEOUT = 700;
         var FAILSAFE_TIMEOUT = 2500;
-
```
Comment (mike-chowla) [inline integrationExamples/gpt/gdpr_hello_world.html:85]: gdpr_hello_world.html shouldn't be included in this commit.  Please revert the changes to it

Comment (Galphimbl) [reply integrationExamples/gpt/gdpr_hello_world.html:85]: i revert changes, but git show that i remove line and than end line again. it's a problem ?
